### PR TITLE
feature: feedback form

### PR DESCRIPTION
**Pull request by kristo-baricevic**
Added Feedback.tsx and FeedbackForm.tsx, changed link in navbar to include feedback endpoint

### CODE REVIEW

File: .DS_Store
Comment (ryanrrogers) [inline .DS_Store:1]: Do you mind adding this to the .gitignore? If you aren't familiar with these, these are generated by macos and generally shouldn't be included.

**Review by ryanrrogers — CHANGES_REQUESTED** [review]
Just throw that .ds_store file in the .gitignore and all is good 👍

File: .DS_Store
Comment (kristo-baricevic) [reply .DS_Store:1]: Hey, I’m away from my computer until tonight but I will do this asap unless I get some time to try it on mobile

Comment (kristo-baricevic) [reply .DS_Store:1]: I added it - but I noticed it was already there. I manually deleted .DS_store but am not sure why it was added despite being in gitignore

Comment (ryanrrogers) [reply .DS_Store:1]: Odd, it might be a weird pathing issue with the .gitignore

**Review by ryanrrogers — APPROVED** [review]

**Pull request merged**